### DISCUSSION TITLE
[IntelliSense] Keep Open VSX Extension up to date

### DISCUSSION THREAD

**Discussion by otherpaco** [tailwindlabs/tailwindcss#13937] [Ideas]
Dear Team,

more and more Open Source IDEs/Editors, like VS Codium or Eclipse THEIA IDE (aka Blueprint), are VSX compatible.

They use the open source VSX Registry instead of the license gated VS-Code-Marketplace.

It would be great to make the latest versions of your extension available via the open source VSX registry.

The VSX version is v0.10.2 while VS-Code-Marketplace offers v0.13.9

Would it be possible to add VSX Registry to your release pipeline/workflow?

That would be very nice. Thank you in advance.

At the moment @bradlc seems to be in charge of VSX releases, maybe he can be of help.

**Comment by lts20050703** [comment]
@bradlc Hello, I was wondering if you guys could provide an update on this? It has been over a year since the last OpenVSX release.

**Comment by flx-sta** [comment]
Bumping this too.
It's 2025 now and MS has **purposely** made it harder to download extension from their marketplace as `.vsix` files (by removing the buttons that used to be there).

Latest [Open VSX Registry](https://open-vsx.org/) available version is [`0.10.2`](https://open-vsx.org/extension/bradlc/vscode-tailwindcss). The most recent one is `0.14.2` though...

**Comment by Germain67** [comment]
@bradlc Could you make `0.14.x` available on https://open-vsx.org/ ? :pray:

**Comment by flx-sta** [comment]
It was updated (13h ago)!
🎉🎉🎉
<img width="1094" alt="image" src="https://github.com/user-attachments/assets/efdde2bf-a7db-4ed4-a9f6-e59fb3e8f0a7" />

**Comment by spiyush19** [comment]
@bradlc, can you please udpate it to 0.14.29?
